site/content/blogs/release-v123.en.md

```diff
@@ -41,6 +41,140 @@ Copilot v1.23 brings several new features and improvements:
 
 ## Move misplaced `http` fields in environment manifest (backward-compatible!)
 
+In [Copilot v1.23.0](https://github.com/aws/copilot-cli/releases/tag/v1.23.0), we are fixing the hierarchy
+under the `http` field in environment manifests.
+
+### What is getting fixed, and why?
+Back in [Copilot v1.20.0](https://aws.github.io/copilot-cli/blogs/release-v120/), we released environment manifest,
```

Comment:
same here-- "the environment manifest" or "environment manifests"

Reply:
Added "the" for both here and there!